RadarSDK/RadarAPIClient.m

```diff
@@ -226,7 +226,7 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
     }
 
     RadarTrackingOptions *options = [Radar getTrackingOptions];
-    if (options.syncGeofences) {
+    if (options.syncGeofences || foreground) {
```

Comment:
So I guess coming out of our Slack conversation yesterday, we probably don't need this since we'll control it server side? cc @lmeier @jsani-radar

Reply:
Revered.